packages/libs/react-components/src/components/Input/styles.ts

```diff
@@ -0,0 +1,130 @@
+/* eslint @kadena-dev/typedef-var: 0 */
+// TODO: Remove this when this issue is resolved: https://github.com/kadena-community/kadena.js/issues/201
+
+import { styled } from '../../styles';
+
+export const styleVariant = {} as const;
+
+export const StyledInputGroupWrapper = styled('div', {
+  color: '$colors$neutral6',
+  '&.disabled': {
+    color: '$colors$neutral3',
+  },
+});
+
+export const StyledInputGroupHeader = styled('div', {
+  display: 'flex',
+  alignItems: 'center',
+  paddingBottom: '8px',
+  '*': {
+    display: 'inline-block',
+  },
+});
+
+export const StyledLabel = styled('label', {
+  fontWeight: '500',
+  fontSize: '14px',
+});
+
+export const StyledTag = styled('span', {
+  marginLeft: '16px',
```

Comment:
can you use the specified spacing and sizing units $1,2,3 etc here and for the rest of the props

Reply:
yes definitly, that was actually one of my questions regard spacings
also I have another issue since in the design actual background color for input is #ffffff, but since I didn't see it's changing regarding light or dark them, I used neutral1 and neutral6 for background and foreground colors, which I hope it's ok